File: taejoong/leetcode/1_TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        int[] arr = new int[2];

        for(int i = 0; i < nums.length - 1; i++) {

            for(int k = i + 1; k < nums.length; k++) {               
                int sum = nums[i] + nums[k];

                if(sum == target) {
                    arr[0] = i;
                    arr[1] = k;
                    return arr;
                }
              }
         }

                return arr;
        }   
    }

File: hosung/leetcode/1. TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        a = []
        n = len(nums)
        for i in range(n):
            for j in range(i+1,n):
                if nums[i]+nums[j] == target:
                    a.append(i)
                    a.append(j)
                    return a
